addon/styles/_frost-pods.scss

```diff
@@ -69,8 +69,14 @@ $frost-color-warn-1: #7D85B2;
 	color: $frost-color-grey-1;
 }
 
+.frost-pods .pod.context > .header .title .primary .comment {
```

Comment:
Hi. could you put all this css into the addon.scss file? Our pattern for singular frost components is to have all the addon styles inside just the one addon file, not a separate file unless there are many components in one addon. Thanks